### PR TITLE
Use server URL

### PR DESCRIPTION
**Pull request by LukasReschke**
Required for the auth changes in core. PR coming there as well soon.

@VicDeo Mind testing whether it still works currently? If yes then please merge and I can finish the core PR :smile:

### CODE REVIEW

**Comment by LukasReschke** [comment]
Basically I do a `jquery.load` in the admin page of ownCloud since we need to set the Authorization header. This means that the URI used will be the one of ownCloud => The API requests failed.

File: src/Resources/views/partials/inner.php [hunk 1]
```diff
@@ -1,4 +1,7 @@
+<link rel="stylesheet" href="<?=$this->uri() . '/pub/' . $this->asset('css/main.css')?>" />
```
Comment (VicDeo) [inline src/Resources/views/partials/inner.php]: @LukasReschke is this really needed here?

File: src/Resources/views/partials/inner.php [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+<script src="<?=$this->uri() . '/pub/' . $this->asset('js/vendor/jquery.min.js')?>"></script>
```
Comment (VicDeo) [inline src/Resources/views/partials/inner.php]: and jQuery too

File: src/Resources/views/partials/inner.php (hunk 1 above)
Comment (LukasReschke) [inline src/Resources/views/partials/inner.php]: Unfortunately yes :disappointed: 

Everything in the head section is ignored when using `jquery.load` as done in https://github.com/owncloud/core/pull/22255. If you have a cleaner solution I'm happy for suggestions :smile:

File: src/Resources/views/partials/inner.php (hunk 2 above)
Comment (LukasReschke) [inline src/Resources/views/partials/inner.php]: Same as https://github.com/owncloud/updater/pull/237#discussion_r52356045

**Comment by VicDeo** [comment]
works, merging :+1:

**Pull request merged**